src/main/scala/io/apibuilder/validation/FormData.scala

```diff
@@ -140,7 +138,7 @@ object FormData {
   }
 
   def toJson(data: Map[String, Seq[String]]): JsObject = {
-    toJson(data.keys.toSeq, data)
+    toJson(data.keys.toSeq.sorted, data)
```

Comment:
@mbryzek Note `.sorted` is the only functional change. Without it, the tests were breaking as the output is not deterministic (order on map keys). I think this determinism is welcome and will even beautify the output string.

Ok to keep?